egs2/ml_superb/asr1/README.md

```diff
@@ -96,6 +96,66 @@ General steps to run tasks in LID trask are as follows:
 ```
 ./run_multi.sh --asr_config <your_training_config> --duration {10min, 1h} --lid true --only_lid false
 ```
+## Adapter usage guidelines
```

Comment:
Would it be better to note that the challenge does not allow the fine-tuning of SSL models?
@ftshijt, what do you think?